src/test/java/edu/ucsb/cs156/gauchoride/controllers/UsersControllerTests.java

```diff
@@ -165,6 +166,91 @@ public void admin_tries_to_delete_non_existant_user_and_gets_right_error_message
           assertEquals("User with id 15 not found", json.get("message"));
   }
 
+  @WithMockUser(roles = { "ADMIN", "USER" })
```

Comment:
The changes here are probably because you didn't start with a fresh copy of the main branch when you made your branch.

It's always good to start with a fresh copy of main *unless* you *need* the changes in that other branch to build your changes on top of. 

In this case, when the only change is adding this code to the User entity:

```
  @Builder.Default
  private boolean driver = false;
```

it would be cleaner and easier to have a PR that makes that change, and ONLY that change.

Reply:
PS: I wonder if you might want to start with a new clean branch, and then just put in that one change?

i.e.
```
git fetch
git checkout main
git pull origin main
git checkout -b sa-add-driver-field-to-user
# make the one change in that file and then do a PR
```

That would be easier to PR because probably the tests would all pass easily.
Then you can build other changes on top of that one.

Reply:
Alright, I'm gathering that I should probably separate these changes into their own branches so it will be easier to test.